tidy3d/components/simulation.py

```diff
@@ -1330,19 +1332,38 @@ def plot(
             Opacity of the monitors. If ``None``, uses Tidy3d default.
         ax : matplotlib.axes._subplots.Axes = None
             Matplotlib axes to plot on, if not specified, one is created.
+        hlim : Tuple[float, float] = None
+            The x range if plotting on y or z planes, y range if plotting on x plane.
```

Comment:
do we want to change to `xy` or `xz` planes, for example? probably clearer like you said.